code/modules/chemistry/tools/emergency_injectors.dm

```diff
@@ -47,11 +50,22 @@
 			else
 				if (!target.reagents)
 					return ..()
-				logTheThing(LOG_COMBAT, user, "injects [constructTarget(target,"combat")] with [src] [log_reagents(src)]")
-				src.reagents.trans_to(target, amount_per_transfer_from_this)
-				user.visible_message(SPAN_ALERT("[user] injects [target == user ? himself_or_herself(user) : target] with [src]!"),\
-				SPAN_ALERT("You inject [target == user ? "yourself" : target] with [src]!"))
-				playsound(target, 'sound/items/hypo.ogg', 40, FALSE)
+				if (src.manipulated_injection)
+					logTheThing(LOG_COMBAT, user, "tries to inject [constructTarget(target,"combat")] with a sabotaged [src] [log_reagents(src)]")
+					user.visible_message(SPAN_ALERT("[user] tries to use [src] on [target == user ? himself_or_herself(user) : target], but [src] fails and sprays its contents on the ground!"),\
+					SPAN_ALERT("You try to use [src] on [target == user ? "yourself" : target], but [src] jams prematurely and all its contents spray onto the ground!"))
+					var/turf/target_turf = get_turf(user)
+					var/datum/reagents/temp_holder = new
+					src.reagents.trans_to_direct(temp_holder, src.amount_per_transfer_from_this)
+					target_turf.fluid_react(temp_holder, temp_holder.total_volume)
+					temp_holder.reaction(target_turf, TOUCH)
+					playsound(user, 'sound/impact_sounds/Generic_Snap_1.ogg', 30, FALSE)
```

Comment:
yeah no, boo copypaste code, please move this stuff to a helper proc so there's not 4 giant copypasted chunks here